static/js/formatters-internal.js

```diff
@@ -264,8 +265,10 @@ export function prettyPrintObject(obj, locale) {
  * @returns {string} The localized affirmative or negative.
  */
 function _prettyPrintBoolean(value, locale) {
-  const language = locale.substring(0,2);
-  switch (language) {
+  const { language, modifier } = parseLocale(locale);
+  const languageAndModifier =  modifier ? `${language}-${modifier}` : language;
```

Comment:
We've used "languageAndModifer" in a few PRs now, although I'm thinking that "writtenLanguage" might be a better wording for it because the modifier simply indicates the script (how the language is written). Although it's probably not worth it to go back and update everything